data/fixtures/test_data.py

```diff
@@ -571,7 +572,7 @@ def add_course(self, num_students=5, num_assignments=1, num_additional_criteria=
         # create a shortcut for first assignment as it is frequently used
         self.assignment = self.assignments[0]
 
-        self.add_answers(num_answers, with_comments=with_comments)
+        self.add_answers(num_answers, with_comments=with_comments, num_non_comparable_ans=num_non_comparable_ans)
```

Comment:
recommend instead of adding it to `add_answers`, just add the new answers in this function below (like draft student)